app/services/PaymentAPIService.scala

```diff
@@ -57,6 +73,26 @@ class PaymentAPIService(wsClient: WSClient, paymentAPIUrl: String) {
       .execute()
   }
 
+  def getPaypalApiError(body: String): Option[PaymentApiError] = {
+    val bodyJson = parse(body)
+    val error = bodyJson.toOption
+    val paypalApiError = error.flatMap(json => decodeErrorWrapper(json.hcursor).toOption)
+    paypalApiError.map(_.error)
+  }
+
+  def showAsSuccessful(response: WSResponse): Boolean = {
```

Comment:
I think `isSuccessful` response is a more idiomatic name

Reply:
Good shout